clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -467,24 +467,25 @@ class P4InfoHelper(object):
         table_entry = p4runtime_pb2.TableEntry()
         table_entry.table_id = self.get_tables_id(table_name)
 
-        for mfn, value in match_fields.items():
-            # A table can contain more than one match filed and in such cases
-            # if one match_type has ternary and other filed is of any type
-            # it is considered as ternary match_type.
-            # In these cases we expect priority field from the user.
-            p4info_match = self.get_match_field(table_name, mfn)
-
-            if p4info_match.match_type == p4info_pb2.MatchField.TERNARY:
-                one_match_type_is_ternary = True
-                # If match_type is ternary, then priority field is expected
-                if priority == None:
-                    print("For ternary match_type, priority field is needed")
-                    sys.exit(1)
-
-            if match_fields:
-                table_entry.match.extend([
-                    self.get_match_field_pb(table_name, mfn, value)
-                ])
+        if match_fields:
+            for mfn, value in match_fields.items():
+                # A table can contain more than one match filed and in such cases
+                # if one match_type has ternary and other filed is of any type
+                # it is considered as ternary match_type.
+                # In these cases we expect priority field from the user.
+                p4info_match = self.get_match_field(table_name, mfn)
+
+                if p4info_match.match_type == p4info_pb2.MatchField.TERNARY:
+                    one_match_type_is_ternary = True
+                    # If match_type is ternary, then priority field is expected
+                    if priority == None:
```

Comment:
`priority is None` ?

In Python, the recommendation is to use `is` instead of `==` when comparing to `None`